testing/ctorrent-dnh/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Contributor: Denis Ryabyy <vv1r0x@gmail.com> 
+# Maintainer: Denis Ryabyy <vv1r0x@gmail.com>
+pkgname=ctorrent-dnh
+pkgver=3.3.2
+pkgrel=3
+pkgdesc="Enhanced CTorrent is a BitTorrent client for unix-like environments"
+url="http://www.rahul.net/dholmes/ctorrent/"
+arch="all"
+license="GPL2"
+
+source="${pkgname}${pkgver}.tar.gz::https://sourceforge.net/projects/dtorrent/files/dtorrent/${pkgver}/${pkgname}${pkgver}.tar.gz/download"
+_builddir="$srcdir"/${pkgname}${pkgver}
```

Comment:
`_builddir` is old convention, it should be named `builddir`. This variable is used e.g. by `default_prepare`.